core/src/main/java/greencity/repository/FactOfTheDayRepo.java

```diff
@@ -1,17 +1,25 @@
 package greencity.repository;
 
 import greencity.entity.FactOfTheDay;
-import java.util.List;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface FactOfTheDayRepo extends JpaRepository<FactOfTheDay, Long> {
     /**
-     * Method finds all{@link FactOfTheDay} with specified name.
-     *
-     * @param name of {@link FactOfTheDay}
-     * @return List of {@link FactOfTheDay}
+     * Method finds {@link FactOfTheDay} that satisfy search query.
+     * @param searchQuery query to search
+     * @return pageable of fact of the day
      */
-    List<FactOfTheDay> findAllByName(String name);
+    @Query(nativeQuery = true, value = "SELECT DISTINCT fd.* FROM fact_of_the_day as fd "
```

Comment:
Why DISTINCT?

Reply:
Because it returns 3 times the same object. It happens because of join SocialNetworkTranslation which have  3 different languages of the same SocialNetwork.ID